module/metrics/execution.go

```diff
@@ -188,15 +188,17 @@ func NewExecutionCollector(tracer module.Tracer, registerer prometheus.Registere
 	blockExecutionTime := prometheus.NewHistogram(prometheus.HistogramOpts{
 		Namespace: namespaceExecution,
 		Subsystem: subsystemRuntime,
-		Name:      "block_execution_time_nanoseconds",
-		Help:      "the total time spent on block execution in nanoseconds",
+		Name:      "block_execution_time_milliseconds",
+		Help:      "the total time spent on block execution in milliseconds",
+		Buckets:   []float64{100, 500, 1000, 1500, 2000, 2500, 3000, 6000},
 	})
 
 	blockComputationUsed := prometheus.NewHistogram(prometheus.HistogramOpts{
 		Namespace: namespaceExecution,
 		Subsystem: subsystemRuntime,
 		Name:      "block_computation_used",
 		Help:      "the total amount of computation used by a block",
+		Buckets:   []float64{1000, 10000, 100000, 500000, 1000000, 5000000, 10000000},
```

Comment:
these buckets seem okay at a first glance, but just a friendly reminder to aim to have all buckets used and to not be TOO granular